source/frontend/src/features/mapSideBar/research/ResearchContainer.tsx

```diff
@@ -117,6 +118,7 @@ export const ResearchContainer: React.FunctionComponent<
     if (formikRef !== undefined) {
       formikRef.current?.setSubmitting(true);
       formikRef.current?.submitForm();
+      setIsValid(formikRef.current?.isValid || false);
```

Comment:
Other code awaits the result of `submitForm()` but this one doesn't. Would it make sense to apply the same pattern here?